Automattic/woocommerce-payments
# Add unit test for Get_Reporting_Payment_Activity.

Fixes https://github.com/Automattic/woocommerce-payments/issues/8779

#### Changes proposed in this Pull Request

Add unit test for [Get_Reporting_Payment_Activity](https://github.com/Automattic/woocommerce-payments/blob/a4c1b31d4a4be6059bf91f31d8829bea1b180a3f/includes/core/server/request/class-get-reporting-payment-activity.php).

#### Testing instructions

* Make sure all unit tests pass.
* Make sure I cover important cases in the newly added unit test `Get_Reporting_Payment_Activity_Test` and I don't have copy-paste errors.

-------------------


**Post merge**
